lib/stylelint-vscode/index.js

```diff
@@ -206,6 +206,12 @@ module.exports = async function stylelintVSCode(textDocument, options = {}, serv
 
 	const stylelint = await resolveStylelint({ ...serverOptions, textDocument });
 
+	if (!stylelint) {
```

Comment:
If stylelint isn't found, should we just silently return no diagnostics, or should we have some way of communicating to the user that no copy of stylelint was resolved?

Reply:
That seems fine to me, if the extension cannot resolve stylelint, getting diagnostics won't be helpful ;)

Reply:
I think it's better to silently return.

However, if this extension can determine the language explicitly specified by the user in the `validate` option, I think it's better to output a log. It works the same as the eslint extension.

https://github.com/microsoft/vscode-eslint/blob/8e5ae72973d02bfb39448f56b0bfeb39ef6fa0e9/server/src/eslintServer.ts#L1106

Reply:
@ota-meshi I added a log statement similar to that of eslint's: https://github.com/stylelint/vscode-stylelint/pull/248#discussion_r720456792